sztp-agent/Dockerfile

```diff
@@ -17,8 +17,8 @@ RUN go build -v -o /opi-sztp-agent && CGO_ENABLED=0 go test -v ./...
 # second stage to reduce image size
 FROM alpine:3.20
 
-RUN apk add --no-cache --no-check-certificate curl && rm -rf /var/cache/apk/*
+RUN apk add --no-cache --no-check-certificate curl dbus networkmanager && rm -rf /var/cache/apk/*
```

Comment:
those additions are not needed I think, I tested basic golang:alpine docker with small script in the issue comment and it worked

Reply:
yes, like the sztp-ci failed with the below error, so thought adding this might solve it, but didn't
```
Error: /30 18:39:11 [ERROR]  failed to get ActiveConnections property: The name org.freedesktop.NetworkManager was not provided by any .service files  
```